index.php

```diff
@@ -19,19 +19,19 @@
 				#search_btn{width:30%;float:right;margin-top:-32px;margin-right:60px;}
 			}
             
-            .side-nav{
-  height:100%;
-  width:250px;
-  position:fixed;
-  z-index:1;
-  top:1;
-  left:1;
-  background-color:#000;
-  opacity: 0.8;
-  padding-top:35px;
-  line-height: 5px;
-     overflow-x:hidden;
-  transition:0.5s;
+			.side-nav {
+    height: 100%;
+    width: 250px;
+    position: fixed;
+    z-index: 1;
+    top: 92px;
+    margin-top: -6px;
+    background-color: #000;
+    opacity: 0.8;
+    padding-top: 35px;
+    line-height: 2px;
+    overflow-x: hidden;
+    transition: 0.5s;
```

Comment:
The CSS works perfectly. Honestly I hard a very hard time myself figuring out how to make look good